solutions/3sum.ts

```diff
@@ -0,0 +1,53 @@
+// 15. 3Sum
+// https://leetcode.com/problems/3sum/
+export default function threeSum(nums: number[]): number[][] {
+  const triplets: [number, number, number][] = [];
+
+  nums.sort((a, b) => a - b);
+
+  for (let i = 0; i < nums.length - 2; ++i) {
+    if (nums[i] > 0) break;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/axross/leetcode-typescript/pullRequest?prid=7152800)
```suggestion
    if (nums[i] > 0) {break;}
```